rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -625,16 +660,59 @@ namespace rsx
 			const f32 x_stride = f32(w) / m_datapoint_count;
 			const u32 tail_index_offset = m_datapoints.size() - m_datapoint_count;
 
-			for (u32 i = 0; i < m_datapoint_count; ++i)
+			for (size_t i = 0; i < m_datapoint_count; ++i)
```

Comment:
whats going on here? this change and the one below it seem pointless...unless I missed something.

Reply:
Some random warning I was getting. It fixed some green wavey underlined text in VS :man_shrugging:

Reply:
It's a good change correctness wise though, especially when indexing arrays it's healthy to use a machine word size data type.